public_api_client/public_api_client/public_voice_client.py

```diff
@@ -0,0 +1,103 @@
+"""provides methods for accessing TRYB's public-voice-api"""
+
+from public_api_client import configuration
+import requests
+from typing import Any, Iterable
+import base64
+import json
+import logging
+
+
+
+SPEECH_TO_TEXT_URL = configuration.tryb_url_prefix + "/public-api/conversions/speech-to-text"
+TEXT_TO_SPEECH_URL = configuration.tryb_url_prefix + "/public-api/conversions/text-to-speech"
+VOICE_ASSISTANT_TEXT_URL = configuration.tryb_url_prefix + "/public-api/voice-assistant/text"
+
+
+
+def _send_request(method: str, url: str, headers: dict[str, str], body: dict[str, Any], stream: bool):
+
+    try:
+        headers["Authorization"] = "Bearer " + configuration.public_api_token 
+        response = requests.request(
```

Comment:
we used `urllib` originally, since it is part of the python standart-library and does not require an additional dependency. However, since its api is simpler, we now use `requests` in newer code.